docs/configuration.md

```diff
@@ -157,13 +157,33 @@ of the most common options to set are:
     or in your default properties file.
   </td>
 </tr>
+<tr>
+  <td><code>spark.driver.memoryOverhead</code></td>
+  <td>driverMemory * 0.10, with minimum of 384 </td>
+  <td>
+    The amount of off-heap memory (in megabytes) to be allocated per driver in cluster mode. This is
+    memory that accounts for things like VM overheads, interned strings, other native overheads, etc.
+    This tends to grow with the container size (typically 6-10%). This option is supported in Yarn
```

Comment:
YARN. I'd also simplify:

"This option is currently supported on YARN and Kubernetes."

Same below.

Reply:
Done.